src/PKSim.Core/Services/MoleculeOntogenyVariabilityUpdater.cs

```diff
@@ -116,16 +118,10 @@ private void updateOntogenyParameterTable(IParameter parameter, TableFormula tab
       {
          //the formula may be null if no ontogeny was selected. In this case, we simply set a default formula of 1
          parameter.Formula = tableFormula ?? _formulaFactory.ValueFor(CoreConstants.DEFAULT_ONTOGENY_FACTOR, parameter.Dimension);
-      }
-
-      private void updateOntogenyParameter(IParameter parameter, double value)
-      {
-         //this can be the case if the parameter is defined in an expression profile
-         if (parameter == null)
-            return;
 
-         parameter.DefaultValue = value;
-         parameter.Value = parameter.DefaultValue.Value;
+         //in case of a tableFormula being set, make sure we also reset the default value of the parameter so that
+         //it will not look like the value was changed by the user
+         parameter.DefaultValue = tableFormula != null ? null : CoreConstants.DEFAULT_ONTOGENY_FACTOR;
```

Comment:
So many issues with this one issue
So here, we need to make sure that the default value is reset for tableFormula otherwise it will be set to the VALUE when exported